dao-contracts/src/reputation.rs

```diff
@@ -134,7 +195,30 @@ impl ReputationContractInterface for ReputationContract {
 
     fn transfer_from(&mut self, owner: Address, recipient: Address, amount: U256) {
         self.access_control.ensure_whitelisted();
-        self.token.raw_transfer(owner, recipient, amount);
+
+        // Load the balance of the owner.
+        let owner_full_balance = self.get_full_balance(owner);
+        let (is_positive_owner, owner_balance) = owner_full_balance;
```

Comment:
I would say the balance is positive not the owner. 😉

Reply:
Done.